installers/osie/main.go

```diff
@@ -119,6 +119,8 @@ func kernelParams(ctx context.Context, action, state string, j job.Job, s ipxe.S
 	switch j.PlanSlug() {
 	case "c2.large.arm", "c2.large.anbox":
 		s.Args("iommu.passthrough=1")
+        default:
+                s.Args("intel_iommu=on amd_iommu=on iommu=pt")
```

Comment:
I don't think `on` is a recognized value for `amd_iommu` (see https://www.kernel.org/doc/Documentation/admin-guide/kernel-parameters.txt), so we should probably take that one out.